index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" type="text/css" href="style-reset.css">
+    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css" crossorigin="anonymous">
+    <title>Cloning Apple</title>
+</head>
+
+<body class='background-color'>
+    <main class="main">
+        <header class="header">
+            <div class="navbar-items">
+                <a><img class="logo-img" alt="svgImg"
+                        src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHg9IjBweCIgeT0iMHB4Igp3aWR0aD0iNTAiIGhlaWdodD0iNTAiCnZpZXdCb3g9IjAgMCAxNzIgMTcyIgpzdHlsZT0iIGZpbGw6IzAwMDAwMDsiPjxnIGZpbGw9Im5vbmUiIGZpbGwtcnVsZT0ibm9uemVybyIgc3Ryb2tlPSJub25lIiBzdHJva2Utd2lkdGg9IjEiIHN0cm9rZS1saW5lY2FwPSJidXR0IiBzdHJva2UtbGluZWpvaW49Im1pdGVyIiBzdHJva2UtbWl0ZXJsaW1pdD0iMTAiIHN0cm9rZS1kYXNoYXJyYXk9IiIgc3Ryb2tlLWRhc2hvZmZzZXQ9IjAiIGZvbnQtZmFtaWx5PSJub25lIiBmb250LXdlaWdodD0ibm9uZSIgZm9udC1zaXplPSJub25lIiB0ZXh0LWFuY2hvcj0ibm9uZSIgc3R5bGU9Im1peC1ibGVuZC1tb2RlOiBub3JtYWwiPjxwYXRoIGQ9Ik0wLDE3MnYtMTcyaDE3MnYxNzJ6IiBmaWxsPSJub25lIj48L3BhdGg+PGcgZmlsbD0iI2VjZjBmMSI+PHBhdGggZD0iTTE1My4xNzQwNiwxMTkuNTRjLTMuNzA4NzUsOC4yMzcxOSAtNS40OTU5NCwxMS45MTkwNiAtMTAuMjY2MjUsMTkuMTg4NzVjLTYuNjc4NDQsMTAuMTU4NzUgLTE2LjA5ODEzLDIyLjg0Mzc1IC0yNy43MzUsMjIuOTI0MzhjLTEwLjM2MDMxLDAuMDk0MDYgLTEzLjAzNDM4LC02Ljc1OTA2IC0yNy4xMDM0NCwtNi42MzgxM2MtMTQuMDU1NjMsMC4wNjcxOSAtMTYuOTg1LDYuNzcyNSAtMjcuMzU4NzUsNi42NjVjLTExLjY1MDMxLC0wLjEwNzUgLTIwLjU1OTM4LC0xMS41MjkzNyAtMjcuMjM3ODEsLTIxLjY3NDY5Yy0xOC42NzgxMywtMjguNDQ3MTkgLTIwLjY1MzQ0LC02MS43OTkwNiAtOS4xMTA2MywtNzkuNTM2NTZjOC4xNywtMTIuNTc3NSAyMS4wODM0NCwtMTkuOTY4MTIgMzMuMjE3NSwtMTkuOTY4MTJjMTIuMzYyNSwwIDIwLjEyOTM4LDYuNzg1OTQgMzAuMzQxODgsNi43ODU5NGM5LjkxNjg3LDAgMTUuOTUwMzEsLTYuNzk5MzggMzAuMjQ3ODEsLTYuNzk5MzhjMTAuODAzNzUsMCAyMi4yMjU2Miw1Ljg4NTYzIDMwLjM5NTYyLDE2LjA0NDM4Yy0yNi43MTM3NSwxNC42NDY4NyAtMjIuMzczNDQsNTIuNzk1OTQgNC42MDkwNiw2My4wMDg0NHpNMTA3LjMxMTg4LDI5LjEzMjVjNS4yMDAzMSwtNi42Nzg0NCA5LjE1MDk0LC0xNi4wOTgxMiA3LjcxMzEyLC0yNS42OTI1Yy04LjQ3OTA2LDAuNTc3ODEgLTE4LjM5NTk0LDUuOTkzMTMgLTI0LjE4NzUsMTMuMDA3NWMtNS4yNTQwNiw2LjM5NjI1IC05LjYwNzgxLDE1Ljg4MzEzIC03LjkxNDY5LDI1LjA2MDk0YzkuMjU4NDQsMC4yOTU2MyAxOC44MzkzNywtNS4yMjcxOSAyNC4zODkwNiwtMTIuMzc1OTR6Ij48L3BhdGg+PC9nPjwvZz48L3N2Zz4=" /></a>
+                <a><span>Store</span></a>
+                <a>Mac</a>
+                <a>iPod</a>
+                <a>iPhone</a>
+                <a>iPad</a>
+                <a>iTunes</a>
+                <a>Support</a>
```

Comment:
- [ ] The `<a>` tags are missing the `href` attribute

Reply:
Added ancor element as suggested